tutormfe/patches/local-docker-compose-prod-services

```diff
@@ -4,3 +4,5 @@ mfe:
     build:
```

Comment:
Could you remove this build context? It is still causing a rebuild on every `tutor local start`, though we don't need it anymore.